config/jobs/kubernetes/sig-scalability/DRA/sig-scalability-presubmit-dra-capz.yaml

```diff
@@ -320,12 +320,16 @@ presubmits:
           value: "3s"
         - name: CL2_LONG_JOB_RUNNING_TIME
           value: "150m"
+        - name: CL2_PROMETHEUS_CPU_SCALE_FACTOR
+          value: "2"
```

Comment:
This will just be 1.2 CPU, I think we need more.

Reply:
Updated in #35222